src/executables/commdb.c

```diff
@@ -485,6 +485,26 @@ process_ha_server_mode (CSS_CONN_ENTRY * conn, char *server_name)
     }
 }
 
+static int
+search_server_pid (const char *server_info, const char *search_pattern)
+{
+  char *p = NULL;
```

Comment:
Build fails. Change the type of p to fix.
```suggestion
  const char *p = NULL;
```